src/main/java/domain/NumberGenerator.java

```diff
@@ -0,0 +1,15 @@
+package domain;
+
+import java.util.Random;
+
+public class NumberGenerator {
+    private static final int RANDOM_NUMBER_UPPER_BOUND = 10;
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private static final Random RANDOM = new Random();
```
상수로 관리하시는거라면 대문자로 처리해주세요!